manifest.json

```diff
@@ -18,6 +18,30 @@
     {
       "path": "images/screenshots",
       "addressable": true
+    },
+    {
+      "path": "main.html",
+      "addressable": true
+    },
+    {
+      "path": "css/main.css",
+      "addressable": true
+    },
+    {
+      "path": "scripts/main.js",
+      "addressable": true
+    },
```

Comment:
As both contribution google-cloud-endpoint-ui and google-cloud-endpoint-type depends on each other (Google contribution has to have UI contribution with it as we are deleting rendering code form VSTS code), I am not getting the significance of using different manifest file. Please brief in case I missed something,

Reply:
It is just a good way to keep the project organized. The other contributions that have support files (so far just build tasks) each have a separate folder and separate *manifest.json*. The separate folders makes it easier to know which file is related to what contribution. The separate *manifest.json* keeps the file paths short, as they are relative to the directory of the partial manifest, and modularizes each contribution, ensuring all information about a contribution can be found inside a single folder. The project build calls `tfx extension create --manifestGlobs **/manifest.json` to combine all of the manifests together.

Reply:
I will try it in next iteration.

Reply:
Fixed it in current iteration